routing/graph.go

```diff
@@ -10,29 +10,29 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
-// routingGraph is an abstract interface that provides information about nodes
-// and edges to pathfinding.
-type routingGraph interface {
-	// forEachNodeChannel calls the callback for every channel of the given
+// Graph is an abstract interface that provides information about nodes and
+// edges to pathfinding.
+type Graph interface {
```

Comment:
Hmm, I remember wanting to do the same rename in an old refactor PR of mine but couldn't due to a naming collision... Maybe we removed the other `Graph` component/struct in the meantime. Or maybe I misremember :sweat_smile: Anyway, this rename makes a lot of sense :+1: